### PR TITLE
Fix cloud-node-manager-windows helm script args

### PR DESCRIPTION
**Pull request by CecileRobertMichon**
#### What type of PR is this?

/kind failing-test

#### What this PR does / why we need it:  

Fixes the Helm chart for Windows clusters (causing failure observed in https://testgrid.k8s.io/provider-azure-cloud-provider-azure#cloud-provider-azure-ccm-windows-capz). The PR that originally introduced this (#3283) was incomplete and did not change the container command and args to use the newly introduced script. This fixes it. Verified locally that it works. 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
```

### CODE REVIEW

File: helm/cloud-provider-azure/templates/cloud-provider-azure.yaml [hunk 1]
```diff
@@ -443,7 +443,16 @@ data:
...
-    Invoke-Expression "$cm --node-name=$env:NODE_NAME --use-instance-metadata=true --kubeconfig=$env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/cpaw/kubeconfig.conf"
+    $argList = @()
+
+    foreach ($var in @("CLOUD_CONFIG", "KUBE_API_BURST", "KUBE_API_CONTENT_TYPE", "KUBE_API_QPS", "MASTER", "MIN_RESYNC_PERIOD", "NODE_STATUS_UPDATE_FREQUENCY", "WAIT_ROUTES")) {
+      if ([System.Environment]::GetEnvironmentVariable($var) -NE $null) {
+        $argList += [System.Environment]::GetEnvironmentVariable($var)
+      }
+    }
+    
+    Write-Host "Running $cm $argList --node-name=$env:NODE_NAME --use-instance-metadata=true --kubeconfig=$env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/cpaw/kubeconfig.conf"
```
Comment (CecileRobertMichon) [inline helm/cloud-provider-azure/templates/cloud-provider-azure.yaml:454]: @marosset do you think it's helpful to keep this for later debugging or should I get rid of it?

Comment (marosset) [reply helm/cloud-provider-azure/templates/cloud-provider-azure.yaml:454]: I always find these messages helpful so my vote is to keep it

**Review by marosset — APPROVED** [review]
/lgtm

**Comment by lzhecheng** [comment]
/approve
Thank you!

**Pull request merged**